client/router_create.go

```diff
@@ -163,11 +163,11 @@ func (cli *VanClient) GetVanControllerSpec(options types.SiteConfigSpec, van *ty
 	annotations := map[string]string{}
 
 	svcs := []*corev1.Service{}
-	if cli.RouteClient != nil {
```

Comment:
Removing this requires setting default appropriately on OpenShift versus k8s. Otherwise, routes will not be created on OpenShift which is current/expected behavior. Comments below.

Reply:
I think your comment is true, but (see below), IMO it is being set appropriatelly, only if the user says "--ingress route" is when routes are created.